corehq/apps/dump_reload/couch/load.py

```diff
@@ -0,0 +1,109 @@
+import json
+from collections import Counter
+
+from couchdbkit.exceptions import ResourceNotFound
+
+from corehq.apps.dump_reload.exceptions import DataExistsException
+from corehq.apps.dump_reload.interface import DataLoader
+from corehq.util.couch import IterDB, get_db_by_doc_type, IterDBCallback, get_document_class_by_doc_type
+
+
+class CouchDataLoader(DataLoader):
+    slug = 'couch'
+
+    def __init__(self, stdout=None, stderr=None):
+        super(CouchDataLoader, self).__init__(stdout, stderr)
+        self._dbs = {}
+        self.success_counter = Counter()
+
+    def _get_db_for_doc_type(self, doc_type):
+        if doc_type not in self._dbs:
+            couch_db = get_db_by_doc_type(doc_type)
+            callback = LoaderCallback(self.success_counter)
+            db = IterDB(couch_db, new_edits=False, callback=callback)
+            db.__enter__()
+            self._dbs[doc_type] = db
+        return self._dbs[doc_type]
+
+    def load_objects(self, object_strings, force=False):
+        total_object_count = 0
+        for obj_string in object_strings:
+            total_object_count += 1
+            doc = json.loads(obj_string)
+            db = self._get_db_for_doc_type(doc['doc_type'])
+            db.save(doc)
```

Comment:
do we not need to do something with `_rev` here (or in the dump command)? I'm surprised couch doesn't complain about this if `_rev` is set.